UPGRADE-API-1.11.md

```diff
@@ -22,11 +22,11 @@
         product_image_prefix: 'media/image'
     ```
 
-1. `Sylius\Bundle\ApiBundle\View\CartShippingMethodInterface` and `Sylius\Bundle\ApiBundle\View\CartShippingMethod` have been removed.
```

Comment:
if I see correctly you should leave everywhere `1.` it will be automatically incremented
check here: https://github.com/Sylius/Sylius/pull/12969/files?short_path=5e90e55#diff-5e90e5541000c7473fa704f5cda54bc018bb1ec54da401371ed0cbb918ed4e29

Reply:
shiiieeeett.... PHPStorm auto-change, I hate you